audisp-tacplus.c

```diff
@@ -334,7 +211,7 @@ send_acct_msg(int tac_fd, int type, char *user, char *tty, char *host,
     int retval;
     struct areply re;
 
-    attr=(struct tac_attrib *)tac_xcalloc(1, sizeof(struct tac_attrib));
+    attr=(struct tac_attrib *)xcalloc(1, sizeof(struct tac_attrib));
```

Comment:
>xcalloc [](http://example.com/codeflow?start=30&length=7)

Why change this? Is it a bug fix?

Reply:
This is not a bug, the upstream project using a libpam_tacplus version provide tac_xcalloc method:
https://github.com/daveolson53/pam_tacplus/blob/1daa4e34db9de5cb28538b9af7ef2c213886c655/libtac/lib/xalloc.h

However, the SONiC version of libpam_tacplus not have tac_xcalloc, we can only use xcalloc from glibc.